tests/TestUtilities.py

```diff
@@ -13,16 +13,18 @@ def execute_coala(func, binary, *args, stdout_only=False):
     :param function:    A main function from coala_json, coala_ci module etc.
     :param binary:      A binary to execute coala test
     :param stdout_only: Get the stdout output only, discard stderr
-    :return:            A tuple holding a return value first and
-                        a stdout output as second element.
+    :return:            A tuple holding a return value as first element,
+                        a stdout output as second element and a stderr output
+                        as third element if stdout_only is False.
```

Comment:
Regarding return type, always including stderr is OK with me.  It will be an unused variable in the tuple unpacking.

wrt `stdout_only`, I would go the other way, and also include a `stderr_only`.  These should raise an exception if the ignored stream has any content, showing the unexpected content.